package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
It looks like you committed your entire project. Please remove everything except for the application file (`.md`)